custom-api/templates/new_sale.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Order Form</title>
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/styles.css') }}">
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <script>
+    $(document).ready(function() {
+      $('#sale-form').submit(function(event) {
+        event.preventDefault(); // Prevent the default form submission
+
+        var form = document.getElementById("sale-form");
+        console.log("storeId: " + form.elements.storeId.value);
+        var formData = new FormData(form);
```

Comment:
when I entered the date wrongly formatted, it failed to insert without any errors in UI or in flask server.

Reply:
Define wrongly formated?

Reply:
never mind. I get it. Can fix.